docs/platform/concepts/service_backups.rst

```diff
@@ -167,6 +167,14 @@ Aiven for ClickHouse backups contain database lists, table schemas, table conten
 
     For more information on Aiven for ClickHouse backups, see :ref:`Backup and restore <backup-and-restore>`.
 
+Where do we store the backups?
```

Comment:
Suggest rewording the subheading to "Backup storage location" to align with the article's headings and tone.